src/main/java/core/basesyntax/transactions/TransactionList.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.transactions;
+
+import core.basesyntax.parsers.ParseFromFile;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionList {
+
+    public List<Transaction> getAllTransactions(String filePath) {
```

Comment:
```suggestion
    public Transaction createTransaction(String lineFromFile) {
```

don't call fileReaderService inside this method
let this class have a single responsibility - to get a line from file (that was previously read by FileReaderService)
and to create a Transaction instance out of this line